index.js

```diff
@@ -44,21 +46,32 @@ function GeneratePath(type, keepReferer = false) {
         } else {
           path = cloudFrontDecode(parts[7]);
         }
+        if (!path) return callback();
+
         if (keepReferer && parts[9] && parts[9] !== '-') {
           var referer = parts[9];
         }
-        // get Referer
-        if (path && referer) generatePath.push([path, referer]);
-        else generatePath.push(path);
+
+        const method = parts[5];
+        // get Referer & method
+        if (method && allowedMethods.some((m) => method.includes(m))) {
+          const obj = { path, method };
```

Comment:
Isn't the regex match required here so the method value doesn't have an extra `"`?

Reply:
Thanks for reviewing! For Cloudfront logs, there is no extra `"` before the method so no regex is needed here.